book/src/pong_tutorial/pong_tutorial_02.md

```diff
@@ -0,0 +1,367 @@
+        Paddle {
+            side: side,
+            width: 1.0,
+            height: 1.0,
+        }
+    }
+}
+```
+
+"But that just looks like a regular struct!" you might say. And you're right, 
+here's the special sauce:
+
+```rust,ignore
+impl Component for Paddle {
+    type Storage = DenseVecStorage<Self>;
+}
+```
+
+By implementing `Component` for our `Paddle` struct, and defining the way we'd 
+like that `Component` data stored, we can now add the `Paddle` component to 
+entities in our game. For more on storage types, check out the 
+[Specs documentation][sb-storage].
+
+## Where is the world?
+
+Now it's time to add a new method to our State implementation: `on_start`. 
+Inside this function, we'll `register` our `Paddle` component on the mutable 
+`World` object we're passed by Amethyst's state machine when the game starts up.
+
+```rust,ignore
+fn on_start(&mut self, world: &mut World) {
+  world.register::<Paddle>();
+}
+```
+
+This `World` gets passed around everywhere. It carries with it all the 
+components and resources in our game. Not only the components and resources we 
+create, but the ones the Amethyst engine itself relies on. For instance, in our 
+`main.rs` we added a `RenderBundle::new()` to our game before calling `run()`. 
+That added default rendering components like `Camera`, `Material`, and `Mesh` to 
+the `World`, some of which we'll be using soon.
+
+## Initialise some entities
+
+Now that we have a Paddle component, let's define some paddle entities that 
+include that component and add them to our `World`.
+
+First let's look at our math imports:
+
+```rust,ignore
+use amethyst::core::cgmath::{Vector3};
+use amethyst::core::transform::{LocalTransform, Transform};
+```
+
+Amethyst uses the [cgmath crate][cg] under the hood and exposes it for our use. 
+Today we just grabbed the `Vector3` type, which is a very good math thing to have.
+
+`LocalTransform` and `Transform` are Amethyst ECS components which carry 
+position and orientation information for rendering. `LocalTransform` is relative 
+to a parent if one exists, while `Transform` is global.
+
+Let's also define some constants for convenience:
+
+```rust,ignore
+const PADDLE_HEIGHT: f32 = 0.30;
+const PADDLE_WIDTH: f32 = 0.05;
+const PADDLE_COLOUR: [f32; 4] = [0.0, 0.0, 1.0, 1.0];
+```
+
+Okay, let's make some entities! We'll define an `initialise_paddles` function 
+which will create left and right paddle entities and attach `LocalTransform` 
+components to them to position them in our world.
+
+```rust,ignore
+/// Initialises one paddle on the left, and one paddle on the right.
+fn initialise_paddles(world: &mut World) {
+
+    let mut left_transform = LocalTransform::default();
+    let mut right_transform = LocalTransform::default();
+
+    // Correctly position the paddles.
+    let y = -PADDLE_HEIGHT / 2.0;
+    left_transform.translation = Vector3::new(-1.0, y, 0.0);
+    right_transform.translation = Vector3::new(1.0 - PADDLE_WIDTH, y, 0.0);
```

Comment:
Maybe a reference down to the Camera section for an explanation of why the paddles are positioned at -1.0 and 1.0?